frontend/pages/user-login/user-login.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { FormEvent, useState } from 'react'
+import styles from './user-login.module.css'
+import { EnrollmentCTA, EnrollmentHeader, SubmitButton, TextInput } from '../../shared-components'
```

Comment:
I think this component would be best inside of the compositons/ directory same with registration since the registration & login pages just import them

Reply:
Hmm? I'm sorry I'm not seeing a `compositions/` directory?

Reply:
Resolving as I presume this was about the duplicate `shared-components/` directory (which has now been addressed)

If that was in error, happy to revisit this!